src/mjolnir/ingest_transit.cc

```diff
@@ -140,10 +144,12 @@ uint32_t to_local_epoch_sec(const std::string& dt) {
   return static_cast<uint32_t>(tp.time_since_epoch().count());
 };
 
-// Read from GTFS feed, sort data into the unique tiles they belong to
-std::priority_queue<tile_transit_info_t> select_transit_tiles(const boost::property_tree::ptree& pt) {
+std::string get_origin_id_base(const std::string& stop_id, const std::string feed_name) {
+  return feed_name + "_" + stop_id;
+}
```

Comment:
I also changed the definition of `onestop_id` to be e.g. `"Jena_stopid1" (plus the node_type extension if it was generated, but here we just get the base)